testplan/report/testing/base.py

```diff
@@ -352,6 +361,26 @@ def tags_index(self):
                 *[child.tags_index for child in self])
         return self._tags_index
 
+    @property
```

Comment:
It doesn't really matter but is there a particular reason for this to be a property and not just an attribute set on `__init__`?

Reply:
> It doesn't really matter but is there a particular reason for this to be a property and not just an attribute set on `__init__`?

Because some of the information values would be generated when testplan finished(eg, end time). And also user can add some custom information values.